src/Aliencube.AzureFunctions.Extensions.OpenApi/Extensions/OpenApiSchemaExtensions.cs

```diff
@@ -28,33 +28,42 @@ public static class OpenApiSchemaExtensions
         /// <remarks>
         /// It runs recursively to build the entire object type. It only takes properties without <see cref="JsonIgnoreAttribute"/>.
         /// </remarks>
-        public static OpenApiSchema ToOpenApiSchema(this Type type, NamingStrategy namingStrategy, OpenApiSchemaVisibilityAttribute attribute = null)
+        public static OpenApiSchema ToOpenApiSchema(this Type type, NamingStrategy namingStrategy, int depth = 0, OpenApiSchemaVisibilityAttribute attribute = null)
```

Comment:
Please consider making `depth` the last parameter - a free way of avoiding possible breaking change for a consumers of a public contract.